### ISSUE TITLE
Initial accessibility status for Video is wrong 

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#29]
## Expected Behavior

Initial accessibility status should be **0/6**
## Current Behavior

Initial accessibility status should be **1/6**
![image](https://user-images.githubusercontent.com/993369/76172846-7aa7ab80-6178-11ea-8950-9f89cfca9f08.png)

## Steps to Reproduce

1. Create a video content
2. Select to configure accessibility

## Possible Solution
Apparently, it's calculating accessibility status as valid for long description

**Comment by CAndresH** [comment]
Camila I checked the platform but I did not find the error that you mention, please check now
![image](https://user-images.githubusercontent.com/50594090/77075912-116a3880-69c1-11ea-87b3-426e4b20e893.png)
